src/sage/categories/fields.py

```diff
@@ -458,6 +458,26 @@ def fraction_field(self):
             """
             return self
 
+        def ideal(self, *gens, **kwds):
+            """
+            Return the ideal generated by gens.
+
+            EXAMPLES::
+
+                sage: QQ.ideal(2)
+                Principal ideal (1) of Rational Field
+                sage: QQ.ideal(0)
+                Principal ideal (0) of Rational Field
+            """
+            if len(gens) == 1 and isinstance(gens[0], (list, tuple)):
+                gens = gens[0]
+            if not isinstance(gens, (list, tuple)):
+                gens = [gens]
```

Comment:
I think this cannot happen: on entry `gens` is always a list, and only if it has length 1 and the first entry is a list (or tuple), we modify it.

Reply:
oui, en effet. J'ai viré ces deux lignes.